example/package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "example",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  },
+  "devDependencies": {
+    "@yourssu/eslint-config": "workspace:*",
+    "@yourssu/prettier-config": "workspace:*",
+    "@types/react": "^18.2.66",
+    "@types/react-dom": "^18.2.22",
+    "@vitejs/plugin-react-swc": "^3.5.0",
```

root 경로에 react가 이미 devDependencies로 명시되어 있는데, 또 설치해줄 필요가 없는 것 같아요
```suggestion
  },
  "devDependencies": {
    "@yourssu/eslint-config": "workspace:*",
    "@yourssu/prettier-config": "workspace:*",
    "@vitejs/plugin-react-swc": "^3.5.0",
```

---

불필요한 속성들 아직 남아있는 상태인데 전부 지워주세요ㅇ